frog/imports/api/activities.js

```diff
@@ -39,50 +50,50 @@ export const addActivity = (activityType, data = {}, id, groupingKey) => {
   }
 };
 
-export const duplicateActivity = activity =>
+export const duplicateActivity = (activity: ActivityDbT) =>
   Activities.insert({
     ...activity,
     _id: uuid(),
     data: { ...activity.data, name: activity.data.name + ' (copy)' }
```

Comment:
I agree, I just fixed the type annotation without looking deeply at all the functions. Do we even use this one?